blocks/pdp/add-to-cart.js

```diff
@@ -75,16 +75,17 @@ function toggleFixedAddToCart(container) {
  */
 export default function renderAddToCart(block, custom) {
   // extract config options from custom object
-  const { findLocally, findDealer, commercial } = custom;
```

Comment:
We exclude commercial products entirely right now, so `commercial` will always be undefined.